### PR TITLE
ci: add back cd

### PR DESCRIPTION
**Pull request by bwanglzu**
add back CD to push documentation and release to pypi, integrated with semantic release.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,3 +1,6 @@
+[metadata]
+version = 0.5.0
```
[the remark this answers is not in the document]
Comment (jupyterjazz) [reply setup.cfg:2]: Gotcha, 0.4.1 was the last version of the old finetuner.

**Review by jupyterjazz — APPROVED** [review]
LGTM!

**Review by guenthermi — APPROVED** [review]
LGTM, just suggesting an update of the description

File: setup.py [hunk 2]
```diff
@@ -1,29 +1,10 @@
...
-    pass
-
+_name = 'finetuner'
 
 # package metadata
-_description = (
-    'Finetuner allows one to tune the weights of any deep neural network for '
-    'better embeddings on search tasks.'
-)
+_description = 'Task-oriented finetuning for better embeddings on neural search.'
```
[the remark this answers is not in the document]
Comment (bwanglzu) [reply setup.py:7]: no, it is our one-liner tag line decided

**Review by gmastrapas — APPROVED** [review]

**Pull request merged**